test/test_torch.py

```diff
@@ -2901,18 +2901,53 @@ def test_tensor_factory_cuda_type(self):
         self.assertTrue(x.is_cuda)
         torch.set_default_tensor_type(saved_type)
 
+    def test_tensor_randoms_bool(self):
+        expectedShape = (1, 2)
+
+        test = torch.rand(expectedShape, dtype=torch.bool)
+        self.assertEqual(expectedShape, test.shape)
+
+        test2 = torch.rand_like(test, dtype=torch.bool)
+        self.assertEqual(test, test2)
+
+        test = torch.rand(expectedShape, dtype=torch.bool)
+        self.assertEqual(expectedShape, test.shape)
+
+        # capped & clamped random
+        test = torch.tensor([[0, 0]], dtype=torch.bool)
+        test.random_(0, 2)
```

Comment:
this isn't going to effect the test -- in place functions don't typically change the shape of a tensor.

Reply:
Yes, i mostly wanted to check if those calls wont fail. Im adding check for a dtype but not sure which test case/assert would do a better job here.